views.go

```diff
@@ -312,7 +312,11 @@ func (lk *logKeeper) appendLog(w http.ResponseWriter, r *http.Request) {
 // GET /build/{build_id}
 
 func (lk *logKeeper) viewBuild(w http.ResponseWriter, r *http.Request) {
-	w.Header().Add("Access-Control-Allow-Origin", "*")
+	// TODO: EVG-17938 - Check this value against an allow list
```

Comment:
Yikes. Can we do..anything better than cors'ing the world?

Reply:
We already were cors'ing the world only difference is now we are cors'ing the world slightly less incorrectly 😞 

We need to add these headers in order to allow requests from parsley (The spiritual successor to lobster) to go through.  This wasn't necessary before because logkeeper hosted its own version of lobster and we didn't have to deal with cross origin requests. But now that we are hosting parsley on its own subdomain we need to allow these headers to bypass the CORS restrictions.

@johndaniels @ybrill is there an admin panel or something i can configure this too i don't mind adding the allow list here i just don't know if we have an established pattern for this.

Reply:
Done just opened a PR in kernel-tools https://github.com/10gen/kernel-tools/pull/347 to add the env variables.

* Added the allow list to this PR. 
* Checks the allow list against the requesting domain.  
* Added a fallback to the preexisting behavior just in case...

I also bumped the utility package to get `StringMatchesAnyRegex`